README.md

```diff
@@ -1,30 +1,49 @@
-PPSSPP - a fast and portable PSP emulator
-=========================================
+# PPSSPP - *A fast and portable PSP emulator*
 
-Created by Henrik Rydgård
+***Created by [Henrik Rydgård](https://github.com/hrydgard)***
 
-Additional code by many contributors, see the Credits screen
+*PPSSPP is an *HLE*([High Level Emulation](https://en-academic.com/dic.nsf/enwiki/1318672)) emulator. Default settings balance good compatibility and speed.*
+**No BIOS file required to play.**
 
-Originally released under the GPL 2.0 (and later) in November 2012
+## 📲 Socials
 
-Official website:
-https://www.ppsspp.org/
+[Official Website](https://www.ppsspp.org)
 
-Discord:
-https://discord.gg/5NJB6dD
+[Discord](https://discord.gg/5NJB6dD)
 
-No BIOS file required to play, PPSSPP is an "HLE" emulator.  Default settings balance good compatibility and speed.
+## 🤝 Contribution
 
 To contribute, see [the development page](https://www.ppsspp.org/development.html).  Help testing, investigating, or fixing is always welcome.  See [the list of issues](https://github.com/hrydgard/ppsspp/issues).
 
 For the latest source code, see [our GitHub page](https://github.com/hrydgard/ppsspp).
 
+## 🔧 Building
+
 For build instructions and other development tutorials, see [the wiki](https://github.com/hrydgard/ppsspp/wiki).
 
-If you want to download regularly updated builds for Android, Windows x86 and x64, proceed to this [page](https://buildbot.orphis.net/ppsspp/)
+If you want to download regularly updated builds for Android, Windows x86 and x64, proceed to this [page](https://buildbot.orphis.net/ppsspp/).
+
+## 🕹️ Compatibility
 
 For game compatibility, see [community compatibility feedback](https://report.ppsspp.org/games).
 
+## 📜 License
+
+Originally released under the GPL 2.0 (and later) in November 2012.
+
+For more details, please read our [LICENSE](https://github.com/hrydgard/ppsspp/blob/master/LICENSE.TXT).
+
+## ♥️ Credits
```

Comment:
AFAIK these are the credits for the Adhoc support so they should belong to the Adhoc support section.

Also I'm not really a fan of italic in general, but that's just my personal opinion :)

Reply:
my bad, i will change that asap and i know windows notepad can't handle unicode emoticon its ignored